app/assets/stylesheets/_color_var.css.scss

```diff
@@ -0,0 +1,11 @@
+
+$accent-color: #FF384D;
+$bg-color: #F9F9F9;
+$dark-bg: #2A2C33;
+$active-dark: #1C1F23;
+$active-accent: #FF2849;
+$highlight: rgba(255, 56, 77, 0.34);
+$bun: #DC8847;
+$softgrey: #E8E8E8;
+$muteWhite: #f7f7f7;
+$muteBlack: #232422;
```

Comment:
Files should end with a trailing newline<br>Name of variable `muteBlack` should be written in all lowercase letters with hyphens instead of underscores